frontend/test/metabase/scenarios/dashboard/subscriptions.cy.spec.js

```diff
@@ -34,7 +34,7 @@ describe("scenarios > dashboard > subscriptions", () => {
     });
   });
 
-  describe("with email and slack set up", () => {
+  describe("with email and/or slack set up", () => {
```

Comment:
seems like it's really just `with email set up` ?

Reply:
Yes. I didn't want to remove Slack completely in case we want to include it in this same group later. I can remove it if it makes more sense?

Reply:
For example, we _might_ need it for this issue:
https://github.com/metabase/metabase/issues/14494

OTOH, it may be easier to separate email from slack if they require different setup.

Reply:
I just like the description and the code to match up. If Slack gets added in that `before` block we can update the description